internal/commands/split.go

```diff
@@ -10,9 +10,10 @@ var splitOutputDir string
 
 func splitCmd() *cobra.Command {
 	cmd := &cobra.Command{
-		Use:   "split [<app-name>] [-o output]",
-		Short: "Split a single-file application into multiple files",
-		Args:  cli.RequiresMaxArgs(1),
+		Use:     "split [APP_NAME] [-o OUTPUT_DIRECTORY]",
```

Comment:
APP_NAME shouldn't have brackets.

Reply:
Use `--output` instead?

Reply:
Add [OPTIONS]?

Reply:
There aren't any other options for split